app/main/session_parser.py

```diff
@@ -645,7 +678,7 @@ class BrewSessionType(str, MultiValueEnum):
 
 
 def dirty_sessions_since_clean(uid, mtype):
-    brew_session_files = list_brew_session_files(uid)
+    brew_session_files = list_session_files(brew_archive_sessions_path(),uid)
```

Comment:
nit, but did you pep8/flake8? no space after ,

Reply:
I did at one point, likely not after that change. I don't run it all the time due to how long the analysis takes.

Reply:
ah whatever I was there and noticed only specific warnings are failing builds, I added more regarding whitespace, newlines in addition to the existing undefined names